### DISCUSSION TITLE
Provide an option to hide commits in a PR discussion thread

### DISCUSSION THREAD

**Discussion by rom1v** [community/community#49775] [Pull Requests]
### Select Topic Area

Bug

### Body

In a PR with many commits referencing the PR and a lot of force-push, the thread is unusable, we cannot follow any discussion.

For example: https://github.com/Genymobile/scrcpy/pull/3757 (search "hidden items" "Load more")

It is polluted by all the commits which are force-pushed, and by commits which reference the PR in their commit message (which are the same commits force-pushed, or sometimes pushed to another branch).

**Comment by xwilson03** [comment]
Bumping this; a couple of my teams use PR discussion forums as a place to document our issues for future developers and this really irks us.

This request also seems to be mentioned here: https://github.com/orgs/community/discussions/12985
